content/en/blog/_posts/2020-01-10-Remembering-Brad-Childs.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: blog
+title: "Remembering Brad Childs"
+date: 2020-01-10T10:00:00-08:00
+slug: Remembering-Brad-Childs
+---
+
+**Authors:** Paul Morie, Red Hat
+
+Earlier this year, the Kubernetes family lost one of its own. Brad Childs was a
```

Comment:
I’m not a native speaker and this is not clear to me... does `this year` referring to the calendar year (2020), or from this day in last year (2019) to today?

Reply:
It's unfortunate, but this looks like a factual error. “Earlier this year” implies that Brad passed away in the calendar year 2020. However, Bradley Childs actually passed away on 2019-10-27.

Reply:
this was fixed by https://github.com/kubernetes/website/pull/18654